>From acbf28e987bd086772ace3348d02d7e310a66a95 Mon Sep 17 00:00:00 2001
From: Cyril Brulebois <kibi@debian.org>
Date: Tue, 28 Apr 2015 16:59:40 +0200
Subject: [PATCH] Update chdist bash-completion file with the architectures
 currently (as of 2015-04-28) listed on buildd.debian.org and
 buildd.debian-ports.org

Signed-off-by: Cyril Brulebois <kibi@debian.org>
---
 debian/changelog               | 7 +++++++
 scripts/chdist.bash_completion | 4 +++-
 2 files changed, 10 insertions(+), 1 deletion(-)

diff --git a/debian/changelog b/debian/changelog
index a56b5fc..ab9654e 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,10 @@
+devscripts (2.15.5) UNRELEASED; urgency=medium
+
+  * Update chdist bash-completion file with the architectures currently (as
+    of 2015-04-28) listed on buildd.debian.org and buildd.debian-ports.org
+
+ -- Cyril Brulebois <kibi@debian.org>  Tue, 28 Apr 2015 16:58:36 +0200
+
 devscripts (2.15.4) unstable; urgency=medium
 
   [ Dominique Dumont ]
diff --git a/scripts/chdist.bash_completion b/scripts/chdist.bash_completion
index 9803659..53b0fc2 100644
--- a/scripts/chdist.bash_completion
+++ b/scripts/chdist.bash_completion
@@ -9,7 +9,9 @@ _chdist ()
 	   compare-versions compare-bin-versions
 	   grep-dctrl-packages grep-dctrl-sources
 	   list'
-  local archs='i386 amd64 powerpc m68k sparc alpha hppa arm mips mipsel ia64 armel s390 ppc64 sh armeb m32r'
+  # Sync'd with buildd.debian.org and buildd.debian-ports.org on 2015-04-28:
+  local archs="amd64 arm64 armel armhf hurd-i386 i386 ia64 kfreebsd-amd64 kfreebsd-i386 mips mipsel powerpc ppc64el s390 s390x sparc \
+    alpha hppa m68k powerpcspe ppc64 sh4 sparc64 x32"
   local dists=$(ls $HOME/.chdist 2>/dev/null)
 
   COMPREPLY=()
-- 
2.1.4

